examples/diabetes_example/assets/functions/federated_analytics_functions.py

```diff
@@ -0,0 +1,73 @@
+import json
+from collections import defaultdict
+
+import pickle
+
+import numpy as np
+import pandas as pd
+import substratools as tools
+
+
+@tools.register
```

Comment:
I think it can be really useful to document what this decorator is doing, and what the functions parameters and outputs mean